src/commands/HelpCommand.ts

```diff
@@ -1,11 +1,13 @@
-import { EmbedBuilder, Message } from 'discord.js';
-import PrefixCommand from './PrefixCommand.js';
+import { ChatInputCommandInteraction, EmbedBuilder } from 'discord.js';
 import BotConfig from '../BotConfig.js';
+import SlashCommand from './commandHandlers/SlashCommand.js';
 
-export default class HelpCommand extends PrefixCommand {
-	public readonly aliases = ['help'];
+export default class HelpCommand extends SlashCommand {
+	public readonly slashCommandBuilder = this.slashCommandBuilder
+		.setName( 'jira-help' )
+		.setDescription( 'Sends a help message.' );
 
-	public async run( message: Message ): Promise<boolean> {
+	public async run( interaction: ChatInputCommandInteraction ): Promise<boolean> {
 		try {
 			const embed = new EmbedBuilder();
 			embed.setTitle( '<:mojira:821162280905211964> **MojiraBot help** <:mojira:821162280905211964>' )
```

Comment:
With slash commands, the bot appears to have the same restrictions for emojis as regular users. For us that's okay but it becomes a slight issue as soon as we let the bot to be invited to other servers. However when we do that we'll probably rework this command anyway.